### ISSUE TITLE
PrimitiveFailedError for ServerOptions

### ISSUE THREAD

**Issue by dyfer** [supercollider-quarks/cruciallib#2]
Starting in SC 3.11 `ServerOptions.devices` works also on Windows (technically - when SC is built with PortAudio). It will still fail on Linux (i.e. when building with JACK), but I think the error kind has changed. Since you are [checking for a particular error](https://github.com/supercollider-quarks/cruciallib/blob/master/ServerTools/ServerGui.sc#L117) it might be worth testing this on Linux with SC 3.11.

I just wanted to share this, since I came across this when working on `ServerOptions`. I hope this will be helpful and apologies for not being able to test myself ATM.

**Comment by crucialfelix** [comment]
Do you mean that it _is_ now a `PrimitiveFailedError`?

https://github.com/supercollider-quarks/cruciallib/blob/f118bd45dd65597e57988c0ed399c2b52f026d1b/ServerTools/ServerGui.sc#L117

Unfortunately I don't have a linux either (though I would like to).

Thanks for the report!

**Comment by dyfer** [comment]
Just to be clear, I referred to the error that `ServerOptions.devices` would return on Linux (or with JACK). 

I realized I can just build with JACK to test this. Result: indeed, the error now is (still?) a `PrimitiveFailedError`. 
```supercollider
ERROR: Primitive '_ListAudioDevices' failed.
```
I believe what I was confused with was that the error message was different beforehand. However I can't confirm that right now.

Anyway, sorry for the noise. It looks like the current implementation (still) should work fine.
